Public/Src/Engine/Processes/UnSandboxedProcess.cs

```diff
@@ -301,16 +301,24 @@ protected Process CreateAndSetUpProcess()
         {
             Contract.Requires(Process == null);
 
+#if !PLATFORM_OSX

```

Comment:
W
Shouldn't we keep the exe does not exist check for windows?

Reply:
It is kept (here it's `! PLATFORM_OSX` so this executes on Windows)